### ISSUE TITLE
no-unused-vars crash with spread operator and babel-eslint

### ISSUE THREAD

**Issue by dmnd** [aladdin-add/eslint-plugin-autofix#48]
### command 
`./node_modules/.bin/eslint file.js`

### file contents
```js
function f(...a) {}
```

### stacktrace
```
TypeError: Cannot read property 'length' of undefined
Occurred while linting /Users/dmnd/github/flexport/test.js:1
    at Object.problem.fix.fixer [as fix] (/Users/dmnd/github/flexport/node_modules/eslint-plugin-autofix/lib/rules/no-unused-vars.js:92:42)
    at normalizeFixes (/Users/dmnd/github/flexport/node_modules/eslint/lib/util/report-translator.js:176:28)
    at args (/Users/dmnd/github/flexport/node_modules/eslint/lib/util/report-translator.js:278:18)
    at Object.report (/Users/dmnd/github/flexport/node_modules/eslint/lib/linter.js:721:41)
    at Object.value [as report] (/Users/dmnd/github/flexport/node_modules/eslint-rule-composer/lib/rule-composer.js:175:29)
    at Program:exit (/Users/dmnd/github/flexport/node_modules/eslint/lib/rules/no-unused-vars.js:643:33)
    at listeners.(anonymous function).forEach.listener (/Users/dmnd/github/flexport/node_modules/eslint/lib/util/safe-emitter.js:45:58)
    at Array.forEach (<anonymous>)
    at Object.emit (/Users/dmnd/github/flexport/node_modules/eslint/lib/util/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/Users/dmnd/github/flexport/node_modules/eslint/lib/util/node-event-generator.js:251:26)
```

### .eslintrc.yaml
```
plugins:
  - autofix

parser: "babel-eslint"

rules:
  autofix/no-unused-vars: error
```

### versions
eslint 5.16.0
babel-eslint 10.0.1
eslint-plugin-autofix 0.0.7

**Comment by aladdin-add** [comment]
fixed in v0.0.8 🎉

**Comment by dmnd** [comment]
Nice, thank you!
